# config/zed_yaml
# Parameters for Stereolabs ZED camera
---

zed:
    zed_node:
        ros__parameters:
            general:
                camera_model:       'zed'
            depth:
                min_depth:          0.3     # Min: 0.3, Max: 3.0
                max_depth:          20.
